ppa/archive/models.py

```diff
@@ -184,6 +255,13 @@ class DigitizedWork(TrackChangesModel, Indexable):
     #: internal team notes, not displayed on the public facing site
     notes = models.TextField(blank=True, default='',
         help_text='Internal curation notes (not displayed on public site)')
+    #: metadata fields that should be preserved on bulk update because
+    # they have been modified by data editors.
+    protected_fields = ProtectedFlagsField(
+        default=ProtectedFlags,
+        help_text='Fields protected from bulk update because they have been '
+                  'manually edited.'
```

Comment:
I might make this even more specific - not just manually edited, but edited in django admin.

Should probably also mention that it's Hathi-specific.

Reply:
Agreed. I've made it point more specifically to Hathi.